Core/Core/Models/ModuleItemType.swift

```diff
@@ -17,6 +17,7 @@
 //
 
 import Foundation
+import UIKit
```

Comment:
Is this supposed to be there?

Reply:
Yeah I got a flaky build failure in here related to this file referencing `UIImage`. I figured this wouldn't hurt. Shouldn't technically be necessary I don't think.